### PR TITLE
[DX-1647] 5.3.5 Dashboard release notes 

### PR DESCRIPTION
**Pull request by Eopayemi**
### For internal users - Please add a Jira DX PR ticket to the subject!
<br>

[DX-1647](https://tyktech.atlassian.net/browse/DX-1647)

### Preview Link

[Preview](https://deploy-preview-5328--tyk-docs.netlify.app/docs/nightly/product-stack/tyk-dashboard/release-notes/version-5.3)
### Description

Dashboard release notes for 5.3.5

#### Screenshots (if appropriate)
<br>

## Checklist

- [ ] I have added a **preview link** to the PR description.
- [ ] I have reviewed the suggestions made by our AI (PR Agent) and updated them accordingly (spelling errors, rephrasing, etc.)
- [ ] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [ ] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [ ] Make sure you have started *your change* off *our latest `master`*.
- [ ] I **labeled** the PR

[DX-1647]: https://tyktech.atlassian.net/browse/DX-1647?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

___

### **PR Type**
documentation

___

### **Description**
- Added detailed release notes for Tyk Dashboard version 5.3.5, including highlights, upgrade instructions, and dependencies.
- Updated tags in the release notes document to include the latest versions 5.3.4 and 5.3.5.
- Provided a compatibility matrix for Tyk components and third-party tools for version 5.3.5.
- Included download links for Docker images and Helm charts for version 5.3.5.

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Documentation</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>version-5.3.md</strong><dd><code>Add release notes and compatibility details for Tyk Dashboard 5.3.5</code></dd></summary>
<hr>

tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.3.md

<li>Added release notes for Tyk Dashboard version 5.3.5.<br> <li> Updated tags to include versions 5.3.4 and 5.3.5.<br> <li> Included compatibility matrix and third-party dependencies for version <br>5.3.5.<br> <li> Provided upgrade instructions and download links for version 5.3.5.

</details>

  </td>
  <td><a href="https://github.com/TykTechnologies/tyk-docs/pull/5328/files#diff-14b1be0b2206c9ee21cb1002b55361919932e84558f614c4342f05b8eb5431f8">+85/-2</a>&nbsp; &nbsp; </td>

</tr>                    
</table></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**: Comment `/help "your question"` on any pull request to receive relevant information

### CODE REVIEW

**Review by dcs3spp — COMMENTED** [review]
Thanks for creating Release notes for Dashboard 5.3.5 @Eopayemi. Let me know once changelog has been added and I will review

**Review by andyo-tyk — CHANGES_REQUESTED** [review]
Confirmed change log (no changes - just a version bump), moved Release Highlights higher for visibility. Needs the release date and Helm links etc to be completed.

**Review by carolwkc — APPROVED** [review]
Helm chart version to be updated when it is released.

**Review by andyo-tyk — APPROVED** [review]

File: tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.3.md [hunk 1]
```diff
@@ -18,6 +18,89 @@ tags: ["Tyk Dashboard", "Release notes", "v5.3", "5.3.0", "5.3.1", "5.3.2", "5.3
...
+
+
+### Release Highlights
+
+This is a version bump to align with Gateway v5.3.5, no changes have been implemented in this release.
+
+### Breaking Changes
+
+**Attention**: Please read this section carefully.
+There are no breaking changes in this release, however, if moving from a version of Tyk older than 5.3.0 please read the explanation provided with [5.3.0 release]({{< ref "#TykOAS-v5.3.0">}}).
```
Comment (letzya) [inline tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.3.md:36]:
```suggestion

There are no breaking changes in this release, however, if moving from a version of Tyk older than 5.3.0 please read the explanation provided with [5.3.0 release]({{< ref "#TykOAS-v5.3.0">}}).
```

**Review by sharadregoti — APPROVED** [review]

File: tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.3.md [hunk 2]
```diff
@@ -18,6 +18,90 @@ tags: ["Tyk Dashboard", "Release notes", "v5.3", "5.3.0", "5.3.1", "5.3.2", "5.3
 As outlined in the [LTS policy]({{< ref "developer-support/special-releases-and-features/long-term-support-releases" >}}), version 5.3 is a long-term support release with full support available until May 2025. Maintenance support will continue until May 2026. Our next long-term support release will be announced at the end of April 2025.
 
 ---
+
+## 5.3.5 Release Notes
+
+
+### Release Date 24 September 2024
```
Comment (sharadregoti) [inline tyk-docs/content/product-stack/tyk-dashboard/release-notes/version-5.3.md]:
```suggestion
### Release Date 26 September 2024
```

**Review by sharadregoti — APPROVED** [review]

**Pull request merged**